### PR TITLE
Change contributing and pr template

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
## Reasoning behind the pull request
- there were unclear instructions for branches management
  
## Proposed Changes
- added guidelines on how branches should be used

## Testing procedure
- only text/approach review

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -3,15 +3,20 @@
...
+a pull request.
 
-External contributions should happen against the `development` branch.
+For external contributors, the PRs should be targeted towards the `master` branch. A team responsible will instruct 
+the PR owner to re-target it against another branch, in accordance to internal branches management.
 
-Other branches may be used by the team members in accordance to internal decisions.
+tl;dr:
+- fork `elrond-go` and use `master` branch
+- use signed commits
```
Comment (andreibancioiu) [inline .github/CONTRIBUTING.md:18]: Perhaps add a link to:

https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -3,15 +3,20 @@
...
 If you are unfamiliar with the workflow of contributing to github, you can refer to this [this article](https://github.com/firstcontributions/first-contributions/blob/master/README.md)
 
-## Fork & clone this repository
+## External contributions 
 
-The development should happen in a personal fork, cloned on the local machine.
+If you are not part of the team and want to contribute to this repository, you must fork & clone this repository
 
-## Use development branch
+The development should happen in a personal fork, cloned on the local machine. Make sure you use signed commits before opening
```
Comment (andreibancioiu) [inline .github/CONTRIBUTING.md:10]: `Make sure ...` in **bold**, perhaps?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -3,15 +3,20 @@
...
+a pull request.
 
-External contributions should happen against the `development` branch.
+For external contributors, the PRs should be targeted towards the `master` branch. A team responsible will instruct 
+the PR owner to re-target it against another branch, in accordance to internal branches management.
 
-Other branches may be used by the team members in accordance to internal decisions.
+tl;dr:
```
Comment (andreibancioiu) [inline .github/CONTRIBUTING.md:16]:
```
### tl;dr:
```

(or bold)

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -3,15 +3,20 @@
...
+- fork `elrond-go` and use `master` branch
+- use signed commits
+- open a PR against `ElrondNetwork/elrond-go`'s `master` branch
```
Comment (andreibancioiu) [inline .github/CONTRIBUTING.md:19]: Optional rephrasing `against the master branch of ...`.

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (iulianpascalau) [inline .github/CONTRIBUTING.md:10]: :+1:

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -3,15 +3,20 @@
...
 
-The development should happen in a personal fork, cloned on the local machine.
+If you are not part of the team and want to contribute to this repository, you must fork & clone this repository
 
-## Use development branch
+The development should happen in a personal fork, cloned on the local machine. Make sure you use signed commits before opening
+a pull request.
 
-External contributions should happen against the `development` branch.
+For external contributors, the PRs should be targeted towards the `master` branch. A team responsible will instruct
```
Comment (iulianpascalau) [inline .github/CONTRIBUTING.md:13]: :+1:

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -12,3 +12,10 @@
 - 
 - 
 - 
+
+## Pre-requisites
+
+Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
```
Comment (andreibancioiu) [inline .github/PULL_REQUEST_TEMPLATE.md:18]: Is it possible to use a relative link here, perhaps?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-## Description of the reasoning behind the pull request (what feature was missing / how the problem was manifesting itself / what was the motive behind the refactoring)
+## Reasoning behind the pull request
```
Comment (iulianpascalau) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: please also change L6 to
```
## Proposed changes
```
as seen in rc/v1.4.0 branch

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (bogdan-rosianu) [reply .github/CONTRIBUTING.md:18]: added

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (bogdan-rosianu) [reply .github/CONTRIBUTING.md:16]: done (bold)

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (bogdan-rosianu) [reply .github/CONTRIBUTING.md:19]: done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 6 above)
Comment (bogdan-rosianu) [reply .github/PULL_REQUEST_TEMPLATE.md:18]: yes, done

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 7 above)
Comment (bogdan-rosianu) [reply .github/PULL_REQUEST_TEMPLATE.md:1]: done

**Review by andreibancioiu — APPROVED** [review]

**Review by AdoAdoAdo — APPROVED** [review]
